pyproj/crs.py

```diff
@@ -52,6 +52,13 @@ class CRS(_CRS):
     * https://github.com/opendatacube/datacube-core/blob/83bae20d2a2469a6417097168fd4ede37fd2abe5/datacube/utils/geometry/_base.py
     * https://github.com/mapbox/rasterio/blob/c13f0943b95c0eaa36ff3f620bd91107aa67b381/rasterio/_crs.pyx
 
+    Attributes
+    ----------
+    srs: str
+        The string form of the user input used to create the CRS.
+    name: str
+        The name of the projection from `proj_get_name`.
```

Comment:
projection -> CRS ? 
(and I also not sure if mentioning `proj_get_name` is that useful for the typical user)

Reply:
Good points. I will update that.

Reply:
Thoughts on the current iteration?